source/whats-new.txt

```diff
@@ -47,6 +47,11 @@ The {+driver-short+} v4.10 release includes the following new features:
 What's New in 4.9
 -----------------
 
+.. warning:: Breaking Changes
+
+  {+driver-short+} v4.9 contains breaking changes. For more information, see
+  :ref:`version-4.9-breaking-changes`.
```

Comment:
```suggestion
  {+driver-short+} v4.9 contains breaking changes. For more information, see
  :ref:`version-4.9-breaking-changes` in the Upgrade {+driver-short+} Versions guide.
```

Reply:
I think for consistency with the other breaking changes warnings I will leave this?